data/priyankarpal.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "Priyankar Pal",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Avid Linux User **Web Dev**. Tech Enthusiast",
+  "avatar": "https://github.com/SaraJaoude.png",
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/priyankarpal",
+      "icon": "FaTwitter"
+    },
+      {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/priyankarpal",
+      "icon": "FaGithub"
+    },
+       {
+      "name": "LinkedIn: Let's connect",
+      "url": "https://linkedin.com/in/priyankarpal",
+      "icon": "FaLinkedin"
+    },
+       {
+      "name": "Instagram: Let's connect",
+      "url": "https://instagram.com/priyankarpal1",
+      "icon": "FaInstagram"
+    },
```

```suggestion
    }
```
There was an extra comma . You can commit this change